### ISSUE TITLE
Administrator Dashboard

### ISSUE THREAD

**Issue by tothegills** [OctopusDeploy/Specs#42]
As an Octopus administrator I would like an overview of the state of the Octopus Server, alerts to any potential issues and how to resolve them.

## Information:

- Counts of stuff (environments, targets, projects, lifecycles, channels, tenants, users, releases, deployments)
- Who is using Octopus right now

## Troubleshooting:

- Are there a large number of releases/deployments that are not being cleaned up by retention policy?
- Are there a large number of packages not being cleaned up by retention policy?
- Are there transactions/requests taking an especially long time?
- Are there tasks taking an especially long time?
- What is the health of the HA cluster (are nodes changing leaders frequently)?
- What is the health of the Octopus database (schema, indexes, statistics)?

## Solutions:

- Configure retention policy
- Perform database maintenance

**Comment by matt-richardson** [comment]
A few things I can think of:
* configure log levels
* see where things are installed
* restart the server
* take a process dump
* visualise the request/task metrics we're capturing
* cpu/memory usage (similar to teamcity)

TeamCity's diagnostics page has this:
![image](https://user-images.githubusercontent.com/373389/40208472-4a63a4fa-5a7d-11e8-8216-3bb72a2d97ef.png)

**Comment by RyPhi76** [comment]
Yes!! Would love to see how many users are logged!! Even have ability to kill off sessions as needed.  I can see all of the resource utilization right from the server (or Zenoss) and would prefer not have the urge to validate Octopus stats against performance monitor.  

But, I have employed the API driven reporting features and they can be super handy dandy, but its a lot of  extra work to get setup initially. It would be neat to have customizable reporting suite/dashboard in-built to the UI to see "Counts of stuff" and slice them by various factors (i.e. Deployments by Success Fail, cancelled, etc).  

Let's do this!!
